src/instance/InstanceClearOptions.ts

```diff
@@ -0,0 +1,9 @@
+import {ApiProperty} from '@nestjs/swagger';
+import {IsNotEmpty, IsUUID} from 'class-validator';
+
+export class InstanceClearOptions {
+  @ApiProperty()
+  @IsNotEmpty()
```

Comment:
Please test if you still need `@IsNotEmpty()` when using `@IsUUID('4')`. A valid UUIDv4 might not be empty so I guess this check is included but please test first.